docs/validators/node-upgrades.md

```diff
@@ -39,8 +39,9 @@ activation epoch will make the nodes proceed with the updated versions of the co
 Since the height of the first block in an epoch isn't deterministic (due to possible roll-backs), the Network Height
 where a feature becomes effective cannot be calculated.
 
-However, the time when a new feature of a bugfix becomes effective is deterministic, as epochs have fixed lengths in rounds.
-Currently, Elrond Mainnet has epochs of `14,400` rounds and a round is `6 sec`. This results in a `24h` epoch.
+However, the time when a new feature of a bugfix becomes effective can be calculated, as epochs have fixed lengths in rounds.
+Currently, Elrond Mainnet has epochs of `14,400` rounds and a round is `6 sec`. This results in a `24h` epoch. However,
+there can delays of a few rounds, due to roll-backs.
```

Comment:
there can be delays *
and due to rollbacks of the start of epoch metablocks

Reply:
done